internal/eventstore/models/search_query.go

```diff
@@ -33,6 +35,10 @@ func (q *SearchQuery) AggregateIDFilter(id string) *SearchQuery {
 	return q.setFilter(NewFilter(Field_AggregateID, id, Operation_Equals))
 }
 
+func (q *SearchQuery) AggregateIDsFilter(ids ...string) *SearchQuery {
+	return q.setFilter(NewFilter(Field_AggregateID, ids, Operation_In))
+}
+
```

Comment:
what is this used for?

Reply:
validation query in org member. 
```
	validationQuery := es_models.NewSearchQuery().
		AggregateTypeFilter("org", "user").
		AggregateIDsFilter(member.AggregateID, member.UserID)
```

Reply:
ah now i get id... couldn't get a reference even ind my IDE... for what reason ever